pyplume/__init__.py

```diff
@@ -0,0 +1,2 @@
+from .mech import *
```

Comment:
I don't think it's generally a good idea to use `import *`, because it clutters the namespace. Instead, maybe just use `from . import mech`, and then precede all functions with `mech.`